PluginGenerator.sln

```diff
@@ -30,24 +25,20 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SonarQube.Plugins.Test.Comm
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SonarQube.Plugins.Roslyn.RoslynPluginGeneratorTests", "Tests\RoslynPluginGeneratorTests\SonarQube.Plugins.Roslyn.RoslynPluginGeneratorTests.csproj", "{AAE84E64-5977-44D7-8959-80F59FA2B0C1}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ExampleAnalyzer2", "Tests\ExampleAnalyzers\ExampleAnalyzer2\ExampleAnalyzer2\ExampleAnalyzer2.csproj", "{AAAFF8DB-1438-4066-BE44-353B09CF6518}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SonarLint.Metadata", "SonarLint.Metadata\SonarLint.Metadata.csproj", "{FD1E676B-A621-494B-9E2F-083940100415}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "IntegrationTests", "Tests\IntegrationTests\IntegrationTests.csproj", "{CC8197AA-D03C-42DB-A71C-5AEEEE0D9FDD}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RoslynAnalyzer11", "Tests\ExampleAnalyzers\RoslynAnalyzer11\RoslynAnalyzer11\RoslynAnalyzer11.csproj", "{B43B3D94-F457-4BBA-A0D2-EF460F43B906}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RoslynAnalyzer10", "Tests\ExampleAnalyzers\RoslynAnalyzer10\RoslynAnalyzer10\RoslynAnalyzer10.csproj", "{AAAFF8DB-1438-4066-BE44-353B09CF6518}"
+EndProject
```

Comment:
ExampleAnalyzer1 and ExampleAnalyzer2 have now been renamed to RoslynAnalyzer11 and RoslynAnalyzer10, respectively, to more accurately reflect that they now test different versions of Roslyn (before they both ran on Roslyn 1.0).